### PR TITLE
api diff between netstandard2.1 and net5.0 shared framework ref

### PR DESCRIPTION
**Pull request by Anipik**
Api diff between preview 5 shared framework and netstandard2.1 ref pack

### CODE REVIEW

File: release-notes/5.0/netstandard2.1/5.0-preview5_Microsoft.Win32.SafeHandles.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Microsoft.Win32.SafeHandles
+
+``` diff
+ namespace Microsoft.Win32.SafeHandles {
+     public sealed class SafeProcessHandle : SafeHandleZeroOrMinusOneIsInvalid {
+-        public override bool IsInvalid { get; }
```
Comment (danroth27) [inline release-notes/5.0/netstandard2.1/5.0-preview5_Microsoft.Win32.SafeHandles.md:6]: Is this a breaking change?

File: release-notes/5.0/netstandard2.1/5.0-preview5_System.Net.WebSockets.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# System.Net.WebSockets
+
+``` diff
+ namespace System.Net.WebSockets {
+     public abstract class WebSocket : IDisposable {
++        public static bool IsApplicationTargeting45();
```
Comment (danroth27) [inline release-notes/5.0/netstandard2.1/5.0-preview5_System.Net.WebSockets.md:6]: WAT? :smile:

File: release-notes/5.0/netstandard2.1/5.0-preview5_Microsoft.Win32.SafeHandles.md (hunk 1 above)
Comment (Anipik) [reply release-notes/5.0/netstandard2.1/5.0-preview5_Microsoft.Win32.SafeHandles.md:6]: I think its a mistake. this is the change that removed it https://github.com/dotnet/runtime/commit/4fed6ab8818fd33298f8ec1cfbe6ffe2387d4637
@ericstj do we need to add this back ?

File: release-notes/5.0/netstandard2.1/5.0-preview5_System.Net.WebSockets.md (hunk 2 above)
Comment (Anipik) [reply release-notes/5.0/netstandard2.1/5.0-preview5_System.Net.WebSockets.md:6]: sry whts the question here ?. Api is showing here as its absent from netstandard2.1

Comment (danroth27) [reply release-notes/5.0/netstandard2.1/5.0-preview5_System.Net.WebSockets.md:6]: I just found it strange that we have a static API on WebSocket for detecting .NET 4.5.

Comment (ericstj) [reply release-notes/5.0/netstandard2.1/5.0-preview5_System.Net.WebSockets.md:6]: https://github.com/dotnet/standard/issues/111

File: release-notes/5.0/netstandard2.1/5.0-preview5_Microsoft.Win32.SafeHandles.md (hunk 1 above)
Comment (ericstj) [reply release-notes/5.0/netstandard2.1/5.0-preview5_Microsoft.Win32.SafeHandles.md:6]: It's not breaking, it's an override.  Code compiled against the old definition would still work at runtime and just call the base.

**Comment by ericstj** [comment]
Merge at will once ready @Anipik

**Comment by Anipik** [comment]
Merged this one as i verified the changes with already approved changes, and some of them by myself
still adding the area-owners 

cc @safern @tarekgh @krwq @carlossanlop @jozkee @AaronRobinsonMSFT @jkoritzinsky @layomia @steveharter @layomia  @tommcdon  @buyaa-n  @GrabYourPitchforks @dotnet/ncl @tannergooding

**Pull request merged**